src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,114 @@
+import cn from 'classnames';
+import { FC, memo } from 'react';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink';
+import { SortLink } from '../SortLink';
+
+type Props = {
+  people: Person[];
+  selectedPerson: string | number;
```

Comment:
it's not clear why person variable contains not person, but some string or number